tests/objc-cli/Makefile

```diff
@@ -52,3 +52,13 @@ test-xctest-leaks: ../leaktest/bin/Debug/leaktest.exe $(MANAGED_DLL) libmanaged.
 
 libLeakCheckAtExit.dylib: leak-at-exit.c
 	clang -arch i386 -arch x86_64 -shared $< -o $@
+
+test-static: test-static-macos test-static-ios test-static-tvos test-static-watchos
+
+test-static-%: managed.dll $(OBJC_GEN)
+	/Library/Frameworks/Mono.framework/Versions/Current/Commands/mono --debug $(OBJC_GEN) --debug managed.dll -c --outdir=build/$@-temp-dir --target=staticlibrary --platform=$*
```

Comment:
Yes, more changes are needed for that, but I wanted to get this in since other people were running into issues I'd already fixed here (-o didn't work).